lib/serializers/item.js

```diff
@@ -296,9 +296,9 @@ const fromMarcJson = async (object, datasource) => {
     }
     // Add recap customer code
     let customerCode = null
-    if (object.isNyplRecord()) {
+    if (object.isNyplRecord() && object.location.code.startsWith('rc')) {
```

Comment:
Should check for presence of `object.location` and `object.location.code` before proceeding as there may be cases where it's missing (e.g. electronic items)